src/dsql/BoolNodes.cpp

```diff
@@ -1611,7 +1678,10 @@ DmlNode* RseBoolNode::parse(thread_db* tdbb, MemoryPool& pool, CompilerScratch*
 	if (csb->csb_currentDMLNode)
 		node->ownSavepoint = false;
 
-	return node;
+	if (blrOp == blr_ansi_all)
+		return FB_NEW_POOL(pool) NotBoolNode(pool, node);
+	else
+		return node;
 }
```

Comment:
Since types are different here, it's not possible to use the ternary.

I never knew that we had a standard code style to not write return-else-return :)

Reply:
I remember Claudio removing unneeded "else"s across the whole codebase. It would be inappropriate to introduce them back ;-)